extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/logging/CustomFormatRequestLoggingHandler.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.vertx.http.runtime.logging;
+
+import java.text.MessageFormat;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.jboss.logging.Logger;
+
+import io.vertx.core.Handler;
+import io.vertx.ext.web.RoutingContext;
+
+public class CustomFormatRequestLoggingHandler implements Handler<RoutingContext> {
```

Comment:
Can we use the `LoggerHandler` returned by one of the static `create` methods exposed instead of this class? See https://github.com/vert-x3/vertx-web/blob/master/vertx-web/src/main/java/io/vertx/ext/web/handler/LoggerHandler.java#L54